ripple/src/main/webapp/script.js

```diff
@@ -0,0 +1,8 @@
+// Display an alert containing the inputted address if user presses enter
+function searchAddress(e){
+	addressInput = document.getElementById("address-input").value;
```

Comment:
maybe a Javascript thing I am not familiar with, but shouldn't this and the line after the if block be indented to match the if block?